### PR TITLE
feat: adds additional `mssql` feature checks

### PR DESCRIPTION
**Pull request by shane-borden**
* Add logic to pull additional data so that unsupported features can be detected by the rules engine
* Allow the perfmon collection interval to be configurable and default to 10 min instead of 1 min.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -106,7 +106,7 @@
   "python.analysis.autoImportCompletions": true,
   "python.analysis.autoFormatStrings": true,
   "python.analysis.extraPaths": ["${workspaceFolder}/src/server"],
-  "editor.formatOnSave": true,
+  "editor.formatOnSave": false,
```
Comment (cofin) [inline .vscode/settings.json:109]: Any reason to change this?

Comment (shane-borden) [reply .vscode/settings.json:109]: Wasnt aware that I did.  I will revert

Comment (cofin) [inline .vscode/settings.json:109]:
```suggestion
  "editor.formatOnSave": true,
```

**Review by cofin — APPROVED** [review]
lgtm

**Pull request merged**